includes/options/class-amp-options-menu.php

```diff
@@ -138,9 +138,11 @@ public function add_menu_items() {
 	 * @since 1.0
 	 */
 	public function render_theme_support() {
-		$theme_support      = AMP_Options_Manager::get_option( 'theme_support' );
-		$native_description = __( 'Reuses active theme\'s templates to display AMP responses but does not use separate URLs for AMP. This means your site is <b>AMP-first</b> and your canonical URLs are AMP.', 'amp' );
-		$paired_description = __( 'Reuses active theme\'s templates to display AMP responses, but uses separate URLs for AMP. Each canonical URL may have a corresponding AMP URL, if the content is fully AMP valid.', 'amp' );
+		$theme_support = AMP_Options_Manager::get_option( 'theme_support' );
+
+		$native_description       = __( 'Reuses active theme\'s templates to display AMP responses but does not use separate URLs for AMP. This means your site is <b>AMP-first</b> and your canonical URLs are AMP.', 'amp' );
+		$transitional_description = __( 'Reuses active theme\'s templates to display AMP responses, but uses separate URLs for AMP. Each canonical URL may have a corresponding AMP URL, if the content is fully AMP valid.', 'amp' );
+		$reader_description       = __( 'Generates AMP content using simplified templates, which are light but may not match the look-and-feel of your site.', 'amp' );
```

Comment:
I think this description should perhaps note something about classic, like “These include the classic templates.”

This also should note that only posts/pages can be served as AMP in this mode. It is not suitable for other kinds of queries.